pkg/invariants/testframework/alertanalyzer/invariant.go

```diff
@@ -25,6 +27,9 @@ func (w *alertSummarySerializer) StartCollection(ctx context.Context, adminRESTC
 
 func (w *alertSummarySerializer) CollectData(ctx context.Context, storageDir string, beginning, end time.Time) (monitorapi.Intervals, []*junitapi.JUnitTestCase, error) {
 	intervals, err := fetchEventIntervalsForAllAlerts(ctx, w.adminRESTConfig, beginning)
+	clientEventIntervals, err2 := clientview.FetchEventIntervalsForRestClientError(ctx, w.adminRESTConfig, beginning)
```

Comment:
These are not conceptually linked to the alert analyzer tests right? How hard is it to break these out into their own InvariantTest? You just need an adminRESTConfig, should be pretty easy.

Reply:
Right, its best to be moved its own InvariantTest